guides/common/modules/con_understanding-package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,32 @@
+[id="understanding_package_mode_and_image_mode_hosts_{context}"]
```

Comment:
```suggestion
[id="understanding-package-mode-and-image-mode-hosts"]
```

suggestion (non-blocking): Align the anchor with the file name & drop context. Using the context attribute often makes it harder to maintain links. If anchor and file name use dashes, then it's ever easier to find things in the repoitory.